### PR TITLE
rfc5280: Populate the trusted store in `chain-untrusted-root`

### PR DESCRIPTION
**Pull request by tetsuo-cpp**
From https://github.com/trail-of-forks/cryptography/pull/1

### CODE REVIEW

**Comment by tetsuo-cpp** [comment]
At the moment, this test just bounces off a check in our API for empty trusted stores (which seems reasonable as you can't do much without any trusted certs). Let's add a certificate to exercise the logic that we want.

File: limbo/testcases/rfc5280.py [hunk 1]
```diff
@@ -396,7 +401,7 @@ def intermediate_ca_without_ca_bit(builder: Builder) -> None:
     leaf = ee_cert(intermediate)
 
     builder = builder.server_validation()
-    builder.trusted_certs().untrusted_intermediates(intermediate).peer_certificate(leaf).fails()
+    builder.trusted_certs(root).untrusted_intermediates(intermediate).peer_certificate(leaf).fails()
```
Comment (tetsuo-cpp) [inline limbo/testcases/rfc5280.py:404]: From what I can tell, these omissions are accidental (we're not testing an untrusted root scenario) so I've just added them back in.

Comment (woodruffw) [reply limbo/testcases/rfc5280.py:404]: Yep, good call.

**Review by woodruffw — APPROVED** [review]
LGTM! I'll deconflict here.

**Pull request merged**